03_launch_mgmt_cluster.sh

```diff
@@ -319,7 +266,7 @@ function update_capm3_imports(){
 }
 
 #
-# Update the images for the CAPM3 deployment file to use local ones
+# Update the images for CAPM3 and BMO deployment file to use local ones
 #
 function update_component_image(){
```

Comment:
since what this function is changing the registry not the image, should not we change the name of this function to something like `set_localregistry()` so it is more about registry not image

Reply:
You are correct in that it used to change only the registry. But this PR makes it actually change everything, so we can set a new image tag if we want. This will be used later to do upgrade tests for BMO.